src/datasource.ts

```diff
@@ -385,6 +392,149 @@ export default class CogniteDatasource extends DataSourceApi<
   fetchSingleAsset = (id: IdEither) => {
     return fetchSingleAsset(id, this.connector);
   };
+  async getRelationshipsDropdowns(
+    refId: string,
+    selector
+  ): Promise<RelationshipsSelectableValue[]> {
+    const settings = {
+      method: HttpMethod.POST,
+      data: {
+        filter: {},
+        limit: 1000,
+      },
+    };
+    try {
+      const response = await this.connector.fetchItems({
+        ...settings,
+        path: `/${selector.type}/list`,
+      });
+      return response.map(({ name, ...rest }) => ({
+        value: rest[selector.keyPropName],
+        label: name,
+      }));
+    } catch (error) {
+      handleError(error, refId);
+      return [];
+    }
+  }
+
+  fetchRelationshipsTargets = async (
+    targets: CogniteQuery[],
+    [min, max]
+  ): Promise<MutableDataFrame[]> => {
+    return Promise.all(
+      targets.map(async (target) => {
+        const { tab, refId, relationsShipsQuery } = target;
+        if (tab === Tab.Relationships) {
```

Comment:
@KissSpinobleZsolt you need to properly group targets on a higher level. so they don't appear here
 see example here
 
 https://github.com/cognitedata/cognite-grafana-datasource/blob/73da5dd283e9be683e13cc2ba4c1f25c08178f93/src/datasource.ts#L614